scripts/amd/gemm/tune_gemm.sh

```diff
@@ -19,4 +19,4 @@ reduceSpace=$5
 
 DRIVER=$(echo $DRIVER | sed -e "s/matmul_grouped.py/matmul.py/g")
```

Comment:
Can you also update the description and input arg list of this script?

Reply:
added.